dist/oauth2-redirect.html

```diff
@@ -33,7 +33,41 @@
             if (qp.code) {
                 delete oauth2.state;
                 oauth2.auth.code = qp.code;
-                createForm(oauth2.auth, qp).submit();
+                var form = new FormData(createForm(oauth2.auth, qp));
+                var headers = new Headers();
+
+                headers.append("Access-Control-Allow-Origin", "*");
+                fetch(oauth2.auth.schema.get("tokenUrl"), {
```

Comment:
If you post this against echo.ponelat.com, what do you see? The same as if you were use the `<form/>` method?